src/ConversionContext.h

```diff
@@ -16,7 +16,16 @@ namespace drafter {
         refract::Registry registry;
 
     public:
+        bool generateSourceMap;
```

Comment:
I don't think `WrapperOptions` will survive our public API cleaning. So, we can change this when that happens.